mindsdb/integrations/handlers/lindorm_handler/lindorm_handler.py

```diff
@@ -0,0 +1,266 @@
+        """
+
+        renderer = SqlalchemyRender(PhoenixDialect)
+        query_str = renderer.get_string(query, with_failback=True)
+        return self.native_query(query_str)
+
+    def get_tables(self) -> StatusResponse:
+        """
+        Return list of entities that will be accessible as tables.
+        Returns:
+            HandlerResponse
+        """
+
+        query = """
+            SELECT DISTINCT TABLE_NAME, TABLE_SCHEM FROM SYSTEM.CATALOG
+        """
+        result = self.native_query(query)
+        df = result.data_frame
+        df = df[df['TABLE_SCHEM'] != 'SYSTEM']
+        df = df.drop('TABLE_SCHEM', axis=1)
+        result.data_frame = df.rename(columns={df.columns[0]: 'table_name'})
+        return result
+
+    def get_columns(self, table_name: str) -> StatusResponse:
+        """
+        Returns a list of entity columns.
+        Args:
+            table_name (str): name of one of tables returned by self.get_tables()
+        Returns:
+            HandlerResponse
+        """
+
+        need_to_close = self.is_connected is False
+
+        connection = self.connect()
+        cursor = connection.cursor()
+
+        try:
+            query = f"SELECT * from {table_name} LIMIT 5"
+            cursor.execute(query)
+            cursor.fetchall()
+
+            response = Response(
+                RESPONSE_TYPE.TABLE,
+                data_frame=pd.DataFrame(
+                    [(x[0], x[1]) for x in cursor.description],
+                    columns=['column_name', 'data_type']
+                )
+            )
+
+        except Exception as e:
+            log.logger.error(f'Error running query: {query} on the Phoenix Query Server!')
+            response = Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=str(e)
+            )
+
+        cursor.close()
+        if need_to_close is True:
+            self.disconnect()
+
+        return response
+
+
+connection_args = OrderedDict(
+    url={
+        'type': ARG_TYPE.STR,
+        'description': 'The URL to the Phoenix Query Server.'
+    },
+    max_retries={
+        'type': ARG_TYPE.INT,
+        'description': 'The maximum number of retries in case there is a connection error.'
+    },
+    autocommit={
+        'type': ARG_TYPE.BOOL,
+        'description': 'The flag for switching the connection to autocommit mode.'
+    },
+    auth={
+        'type': ARG_TYPE.STR,
+        'description': 'An authentication configuration object as expected by the underlying python_requests and python_requests_gssapi library.'
+    },
+    authentication={
+        'type': ARG_TYPE.STR,
+        'description': 'An alternative way to specify the authentication mechanism that mimics the semantics of the JDBC drirver.'
+    },
+    avatica_user={
+        'type': ARG_TYPE.STR,
+        'description': 'The username for BASIC or DIGEST authentication. Use in conjunction with the authentication option.'
+    },
+    avatica_password={
```

Comment:
This is not used in the code above? Do we need it?

Reply:
Nope we probably don't it was present in the phoenixdb this is simply copy of that just with the lindorm_db_user and lindorm_password

Reply:
Removed the code.